db/migrations/20180809214843_create_ilks.sql

```diff
@@ -7,7 +7,7 @@ CREATE TABLE maker.ilks
 );
 
 COMMENT ON TABLE maker.ilks
-    IS E'@name raw_ilks';
+    IS E'@name raw_ilks\nCDP type';
```

Comment:
Should this say "Collateral type"? Although idk maybe "CDP type" will make more sense to people using the API.